scripts/deployment/lib/deploy-contract.sh

```
@@ -0,0 +1,51 @@
+
+# -- URLs --
+DEPLOY_API_URL="http://127.0.0.1:${DEPLOY_PORT_API}"
+DEPLOY_DEPLOYER_URL="http://127.0.0.1:${DEPLOY_PORT_DEPLOYER}"
+
+# -- Deployer health endpoints --
+DEPLOY_HEALTH_PATH="/health"
+DEPLOY_HEALTH_DETAILS_PATH="/health/details"
+DEPLOY_POKE_PATH="/poke"
+
+# -- Task statuses (deploy.task_status enum) --
+DEPLOY_STATUS_PENDING="pending"
```

Shellcheck (0.11.0)

[warning] DEPLOY_STATUS_PENDING appears unused. Verify use (or export if used externally).

(SC2034)